src/chunks/job-description-chunk/job-description-perksandbenefits-template-chunk/perksandbenefits.tsx

```diff
@@ -11,78 +11,85 @@ import VidIcon from "@assets/icons/VidIcon";
 
 function Perksandbenefits() {
   return (
-    <div className="w-full h-full max-w-[1350px] mt-10 mx-auto mb-6">
+    <div className="w-full h-full max-w-[1350px] mt-10  mb-6 gap-[24px]">
```

Comment:
gap-6 instead, use less of the actual styles [], tailwind is helping you by making it shorter, you should only use [] when you can't find the style in tailwind